src/utils/constants.ts

```diff
@@ -85,4 +85,17 @@ const StorageKeys = {
   TOAST: 'toast',
 };
 
-export {Constants, StorageKeys};
+/**
+ * Local storage keys for intervention impressions.
+ */
+const ImpressionStorageKeys = {
+  CONTRIBUTION: 'impressions_contribution',
```

Comment:
nit: The key string does not need to be completely descriptive, but it would be a little bit beneficial to shorten it. What about using the shorter abbreviation "imp_" instead of "impressions_"?

Reply:
Done, will update DD too.